src/NodaTime/Text/PeriodPattern.cs

```diff
@@ -184,6 +184,14 @@ public StringBuilder AppendFormat(Period value, StringBuilder builder)
             {
                 Preconditions.CheckNotNull(value, nameof(value));
                 Preconditions.CheckNotNull(builder, nameof(builder));
+                // Always ensure we've got *some* unit; arbitrarily pick days.
+                // Note: "P0S" might be nicer here, but the normalizing version picked "P0D"
```

Comment:
I was originally confused as to what "the normalizing version" and "both" (below) referred to; perhaps expand a bit?

Reply:
Done - PTAL. (Will rebase back to a single comment once you're happy with it.)